lib/k8s/stack.rb

```diff
@@ -117,7 +117,8 @@ def keep_resource?(resource)
 
     # Delete all stack resources that were not applied
     def prune(client, keep_resources: )
-      client.list_resources(labelSelector: {@label => name}).each do |resource|
+      # using skip_forbidden: assume we can't create resource types that we are forbidden to list, so we don't need to prune them either
+      client.list_resources(labelSelector: {@label => name}, skip_forbidden: true).each do |resource|
```

Comment:
Maybe this should be a parameter to `stack.prune`, defaulting to true?

Reply:
Yes, makes sense.

Reply:
Done